### PR TITLE
fix: Validation error typo

### PR DESCRIPTION
**Pull request by peterstarling**

### CODE REVIEW

File: packages/model/src/error.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { ErrorObject } from 'ajv';
 
 export type ValidationErrorResponse = {
   error: 'Bad Request';
-  message: 'Validation Error';
+  message: 'Validation error';
```
Comment (niezgoda) [inline packages/model/src/error.ts:5]: it would be easier to have validation error message const

**Review by niezgoda — APPROVED** [review]

File: packages/model/src/error.ts (hunk 1 above)
Comment (peterstarling) [reply packages/model/src/error.ts:5]: to be fair i would have done but i think we want to change the way we throw these errors (see: https://asaphub.atlassian.net/browse/CRN-777) and i think at that point the const will become obsolete - i would imagine we should just use specific error instances? what do you thnink

Comment (peterstarling) [reply packages/model/src/error.ts]: actually i've made the change :D

**Review by niezgoda — APPROVED** [review]

**Comment by peterstarling** [comment]
@kmaid have you tested FE with this change? are we merging it?

File: packages/model/tests/error.test.ts [hunk 2]
```diff
@@ -1,12 +1,15 @@
-import { isValidationErrorResponse } from '../src/error';
+import {
+  isValidationErrorResponse,
+  validationErrorMessage,
```
Comment (kmaid) [inline packages/model/tests/error.test.ts:3]: ???

**Pull request merged**